code/obj/machinery/manufacturer.dm

```diff
@@ -2373,10 +2373,10 @@
 	/datum/manufacture/co2_can,
 	/datum/manufacture/n2_can,
 	/datum/manufacture/plasma_can)
-
-// a blank manufacturer for mechanics
-
-/obj/machinery/manufacturer/mechanic
+	#ifndef RP_MODE
+	hidden = list(/datum/manufacture/agentb_can)
+	#endif
```

Comment:
Maybe I just don't really get arguments as to why, but this still doesn't quite make sense to me as someone doesn't actually play on RP.  The mechanical difference doesn't really benefit the feel/flow/behavior of the game mode as other deviations.  Would the fact that it is a hidden item mean it is "forbidden" and/or "meta knowledge" and thus push it to antagonistic use?  Time for audits! 🧑‍💼 

Ideally the amount of Agent B you get from the raw materials is <= what could be harvested from the material to justify simplification of the process.

Reply:
It being an hidden item doesnt really stop people from using it, considering other hidden items are practically required for effective job use (see botanist hacking fabs for weed, or borgs hacking fabs for expansion upgrades) (Note I think this is fine tbh).

Reply:
Changed